### PR TITLE
Feature send swap ui changes

### PR DESCRIPTION
**Pull request by bradleySuira**

### CODE REVIEW

File: src/views/Send.vue [hunk 1]
```diff
@@ -117,42 +126,44 @@
...
       <div class="wrapper_top form">
         <div class="form-group">
-          <label> Send <span class="text-muted">(INCL FEES)</span> </label>
+          <label>
+            Send <span v-if="includeFees" class="text-muted">(INCL FEES)</span>
+          </label>
           <p class="confirm-value" :style="getAssetColorStyle(asset)">
             {{ amountToSend }} {{ asset }}
           </p>
+          <p v-if="!includeFees">~{{ totalFee }} ETH FEES</p>
```
Comment (monokh) [inline src/views/Send.vue:135]: This is kind of strange. Why only BTC will show fees as included? It's all the same model
The fee is separate from the amount you send

**Review by monokh — CHANGES_REQUESTED** [review]
Overall looking pretty good !! 
A few small issues 

For RBTC getting this on send screen:
image
Fee should be in RBTC

Getting an amount in ETH in the review screen for sending DAI: 
image

Similar issues on Swap. Swapping RBTC - DAI
RBTC Fee shown in 'ETH'

and Receive DAI amount shown in ETH
image


Should the RBTC dollar amount stick to the top?
image

File: src/views/Send.vue [hunk 2]
```diff
@@ -176,146 +187,165 @@
...
-import cryptoassets from '@/utils/cryptoassets'
-import NavBar from '@/components/NavBar'
-import FeeSelector from '@/components/FeeSelector'
-import { prettyBalance, prettyFiatBalance } from '@/utils/coinFormatter'
+import { mapState, mapActions } from "vuex";
+import BN from "bignumber.js";
+import cryptoassets from "@/utils/cryptoassets";
+import NavBar from "@/components/NavBar";
+import FeeSelector from "@/components/FeeSelector";
+import { prettyBalance, prettyFiatBalance } from "@/utils/coinFormatter";
```
[the remark this answers is not in the document]
Comment (bradleySuira) [reply src/views/Send.vue:195]: fixed

File: src/views/Send.vue (hunk 1 above)
Comment (bradleySuira) [reply src/views/Send.vue:135]: Hi @monokh I asked to Thessy and the design proposal shows the fees included only for BTC 
Screen Shot 2020-12-01 at 9 40 32 AM

Comment (monokh) [reply src/views/Send.vue:135]: I think she may be confused something about that... There is no difference in how those are handled, maybe she thought in BTC, fees are taken off what you send.

**Review by monokh — APPROVED** [review]
LGTM

**Pull request merged**